SteamKit2/SteamKit2/Base/Generated/SteamMsgRemoteClient.cs

```diff
@@ -9,33 +9,196 @@
 namespace SteamKit2.Internal
 {
 
-    public interface IRemoteClient
+    public class RemoteClient : SteamUnifiedMessages.UnifiedService
     {
-        CRemoteClient_GetPairingInfo_Response GetPairingInfo(CRemoteClient_GetPairingInfo_Request request);
-        NoResponse NotifyOnline(CRemoteClient_Online_Notification request);
-        CRemoteClient_GetRecentClients_Response GetRecentClients(CRemoteClient_GetRecentClients_Request request);
-        CRemoteClient_MarkTaskComplete_Response MarkTaskComplete(CRemoteClient_MarkTaskComplete_Request request);
-        NoResponse NotifyReplyPacket(CRemoteClient_ReplyPacket_Notification request);
-        CRemoteClient_AllocateRelayServer_Response AllocateRelayServer(CRemoteClient_AllocateRelayServer_Request request);
-        CRemoteClient_AllocateSDR_Response AllocateSDR(CRemoteClient_AllocateSDR_Request request);
-        NoResponse SendSteamBroadcastPacket(CRemoteClient_SteamBroadcast_Notification request);
-        NoResponse SendSteamToSteamPacket(CRemoteClient_SteamToSteam_Notification request);
-        CRemotePlay_SessionStarted_Response SendRemotePlaySessionStarted(CRemotePlay_SessionStarted_Request request);
-        NoResponse SendRemotePlaySessionStopped(CRemotePlay_SessionStopped_Notification request);
-        NoResponse SendRemotePlayTogetherPacket(CRemotePlayTogether_Notification request);
-        CRemoteClient_CreateRemotePlayTogetherInvitation_Response CreateRemotePlayTogetherInvitation(CRemoteClient_CreateRemotePlayTogetherInvitation_Request request);
-        CRemoteClient_DeleteRemotePlayTogetherInvitation_Response DeleteRemotePlayTogetherInvitation(CRemoteClient_DeleteRemotePlayTogetherInvitation_Request request);
+        public override string ServiceName { get; } = "RemoteClient";
+
+        public AsyncJob<SteamUnifiedMessages.ServiceMsg<CRemoteClient_GetPairingInfo_Response>> GetPairingInfo(CRemoteClient_GetPairingInfo_Request request)
+        {
+            return UnifiedMessages.SendMessage<CRemoteClient_GetPairingInfo_Request, CRemoteClient_GetPairingInfo_Response>( "RemoteClient.GetPairingInfo#1", request );
+        }
+
+        public AsyncJob<SteamUnifiedMessages.ServiceMsg<NoResponse>> NotifyOnline(CRemoteClient_Online_Notification request)
+        {
+            return UnifiedMessages.SendMessage<CRemoteClient_Online_Notification, NoResponse>( "RemoteClient.NotifyOnline#1", request );
```

Comment:
Correct me if I misunderstood you, but this request is generated no?

Reply:
It needs to call SendNotification